run/bmv2/run

```diff
@@ -47,7 +47,7 @@ sleep 2
 # Run and show log
 testerRunName=tester-${rnd}
 echo "*** Starting ${testerRunName}..."
-docker run --name ${testerRunName} -it --privileged --rm \
+docker run --name ${testerRunName} $(test -t 0 && echo "-it" || echo "-t") --privileged --rm \
```

Comment:
For maintenability, it would be good to explain why we're testing for stdin. Is my interpretation is accurate?
```suggestion
# Do not attach stdin if running in an environment without it (e.g., Jenkins)
it=$(test -t 0 && echo "-it" || echo "-t")
docker run --name ${testerRunName} ${it} --privileged --rm \
```